medusa/indexers/tmdb/api.py

```diff
@@ -295,6 +295,8 @@ def _get_episodes(self, tmdb_id, specials=False, aired_season=None):  # pylint:
         if not isinstance(episodes, list):
             episodes = [episodes]
 
+        absolute_number_counter = 1
```

Comment:
@medariox what do you think about this to add absolute numbering, when the indexer does not provide it?